torch/__init__.py

```diff
@@ -128,15 +128,18 @@ def is_storage(obj):
 
 
 def set_default_tensor_type(t):
-    global Storage
-    Tensor = _import_dotted_name(t)
-    Storage = _import_dotted_name(t.replace('Tensor', 'Storage'))
+    if isinstance(t, globals()['dtype']):
```

Comment:
What's wrong with `torch.dtype` instead of `globals()['dtype']`?

Reply:
It's because it's the current module, see https://stackoverflow.com/questions/990422/how-to-get-a-reference-to-current-modules-attributes-in-python.